File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/fs/vgmrips/Group.java
package app.zxtune.fs.vgmrips;

import androidx.annotation.NonNull;

/**
 * Defines any grouping entity (chip, company, composer, system etc)
 */
public class Group {
  @NonNull
  public String id;
  @NonNull
  public String title;
  public int packs;

  Group(String id, String title, int packs) {
    this.id = id;
    this.title = title;
    this.packs = packs;
  }
}

File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/fs/vgmrips/Group.kt
package app.zxtune.fs.vgmrips

/**
 * Defines any grouping entity (chip, company, composer, system etc)
 */
data class Group(val id: Id, val title: String, val packs: Int, val image: FilePath? = null) {
    @JvmInline
    value class Id(val value: String)
}
